docs/setup/ubuntu.md

```diff
@@ -1,6 +1,6 @@
 ---
-layout: default
```

Comment:
Any reason why the main setup page has a `page` layout and the OS-specific setup pages have none?